learn/nginx.conf

```diff
@@ -0,0 +1,66 @@
+user user user;
+worker_processes 1;
+error_log /var/log/nginx/error.log;
+pid /dev/null;
+daemon off;
+
+# Load dynamic modules. See /usr/share/nginx/README.dynamic.
+include /usr/share/nginx/modules/*.conf;
+
+env TEST_DATA=/cache;
+env HOME=/home/user;
```

Comment:
Does it actually need a legit home dir? Would /tmp/ do, to avoid having to set up a user and permissions?

Reply:
just removed, works fine without. Copied over a bit too much :)